pkgs/tools/misc/git-town/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }: 
+
+let
+  version = "7.2.0";
+in
+  buildGoPackage rec {
```

Comment:
this let is unused; you define `version` at line 8

Reply:
removed